### PR TITLE
Add welcome message in /etc

### PR DESCRIPTION
**Pull request by KellerFuchs**
Part of hashbang/dotfiles#31, please merge before the dotfiles PR.

### CODE REVIEW

File: hashbang/welcome [hunk 1]
```diff
@@ -0,0 +1,11 @@
+if [ -d "${HB_CONF}" ]; then
```
Comment (daurnimator) [inline hashbang/welcome]: Where does HB_CONF get initialised?

Comment (KellerFuchs) [inline hashbang/welcome]: In the script that sources `hashbang/welcome`, but it's true it is now useless cruft.

File: hashbang/welcome.pre [hunk 2]
```diff
@@ -0,0 +1,9 @@
+   _  _    __
+ _| || |_ |  |  Welcome to #!. This network has three rules:
+|_  __  _||  |
+ _| || |_ |  |  1. When people need help, teach. Don't do it for them
+|_  __  _||__|  2. Don't use our resources for closed source projects
+  |_||_|  (__)  3. Be excellent to each other
+
+ Things to explore:
```
Comment (daurnimator) [inline hashbang/welcome.pre]: I feel like "Things to explore:" might be better duplicated in welcome.{not,t}mux

File: hashbang/welcome.post [hunk 3]
```diff
@@ -0,0 +1,8 @@
+   * To learn more about us and our offerings type: man hashbang
```
Comment (daurnimator) [inline hashbang/welcome.post]: Move empty line to before this?

File: hashbang/welcome.pre (hunk 2 above)
Comment (KellerFuchs) [inline hashbang/welcome.pre]: ACK

File: hashbang/welcome.post (hunk 3 above)
Comment (KellerFuchs) [inline hashbang/welcome.post]: Mmmh, I disagree on that one: currently, all files start with a non-empty line, and I think it's easier to get it right if it's consistent.

**Comment by RyanSquared** [comment]
I don't see how this is triggered, or if there's one somewhere else, a way to not trigger it (for example, `touch ~/.disable-motd`).

**Comment by KellerFuchs** [comment]
@RyanSquared Triggered from dotfiles (see the linked PR), and the way not to trigger it is not to source `/etc/hashbang/welcome`

**Comment by KellerFuchs** [comment]
@RyanSquared I think you put the comment in the wrong PR.  Also, `-n` is definitely required.

**Pull request merged**